Add Python integration docs for Graphene (#8082)
getsentry/sentry-docs: src/platforms/python/common/integrations/graphene/index.mdx

    def resolve_hello(self, info):
        1 / 0
        return "World"

schema = graphene.Schema(query=Query)

schema.execute("""
  query {
    hello
  }
""")
```

We've snuck a `ZeroDivisionError` into our `resolve_hello` resolver. When this
code snippet is run, a new error event will appear in the Issues section of
[sentry.io](https://sentry.io).

It takes a couple of moments for the data to appear in [sentry.io](https://sentry.io).

## Options

When used together with one of our web framework integrations like
[FastAPI](/platforms/python/integrations/fastapi) or
[Flask](/platforms/python/integrations/flask), the Graphene
integration can capture the request body for each GraphQL error that happens.
Since the requests may contain sensitive data, no request bodies will be attached
by default.

To capture request bodies:

- Initialize the SDK with the
  [send_default_pii](/platforms/python/configuration/options/#send-default-pii)
  option set to `True`.
- Make sure the integration for the web framework you're using is enabled.

```python
sentry_sdk.init(
    # same options as above
    send_default_pii=True,
)
```

<Alert level="warning" title="Note">

> Since `send_default_pii` is a global SDK option, setting it to `True` affects all
> integrations, not just Graphene. Please make sure to take care of
> [scrubbing sensitive data](/platforms/python/data-management/sensitive-data/)
> from events before enabling this option.

Reviewer: ```suggestion
Since `send_default_pii` is a global SDK option, setting it to `True` will affect all
integrations, not just Graphene. Please make sure to 
[scrub sensitive data](/platforms/python/data-management/sensitive-data/)
from events before enabling this option.
```
